appveyor.yml

```diff
@@ -87,7 +97,16 @@ before_build:
       else
       {
         $env:SQLITE3_LIBRARY = $env:SQLITE_ROOT + '/lib/sqlite3-static.lib'
-        $env:BUILD_TOOL_OPTIONS = '/m'
+        $env:BUILD_TOOL_OPTIONS = ''
+        # Set up Visual Studio environment (MSVC compiler etc.)
```

Comment:
I wonder why is this needed now when we didn't need this before?

Reply:
Why it wasn't needed before, I don't know but without this, the wrong compiler is selected in the CI